packages/pluggableWidgets/image-viewer-web/tests/e2e/specs/onClick.spec.ts

```diff
@@ -0,0 +1,56 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+import ImageViewer from "../objects/imageViewer.widget";
+import * as html from "../objects/constants";
+
+// eslint-disable-next-line jest/no-disabled-tests
+xdescribe("Image viewer", () => {
+    it("should trigger Microflow on click", () => {
+        page.open("p/onClickMicroflow");
+
+        const imageViewer = new ImageViewer("imageViewer1");
+        imageViewer.element.waitForDisplayed();
+        imageViewer.element.click();
+
+        browser.pause(1000);
```

Comment:
Is this necessary? Could it also work by extending the waitForDisplayed timeout?

Reply:
No, isn't necessary, I'll remove that, when I enable the events tests.